BookOfHabits/BookOfHabitsMicroservice.Domain.Entity/Enums/CoinsOptions.cs

```diff
@@ -13,6 +13,7 @@ public enum CoinsOptions
         FallingLevel = 1,
         AutoPayment = 2,
         CheckingLimit = 4,
-        CheckingReport = 8
+        CheckingReport = 8,
+        All = 15
```

Comment:
why did you setup 15 after 8? It seems like it should be 16 if you use bit flags instead of a standard enum

Reply:
you can use any combination of bits and it looks correct for "All"

Reply:
All - this is an auxiliary field, which value to expect if all flags are enabled.
1 + 2 +4 + 8 = 15